frontend/packages/kubevirt-plugin/src/components/vms/vm.tsx

```diff
@@ -248,6 +269,27 @@ const VirtualMachinesPage: React.FC<VirtualMachinesPageProps> = (props) => {
     },
   ];
 
+  const [importAllowed, setImportAllowed] = useSafetyFirst(false);
+
+  React.useEffect(() => {
+    VMWARE_KUBEVIRT_VMWARE_CONFIG_MAP_NAMESPACES.forEach((configMapNamespace) => {
+      checkAccess({
+        group: ConfigMapModel.apiGroup,
+        verb: 'get',
+        resource: ConfigMapModel.plural,
+        name: VMWARE_KUBEVIRT_VMWARE_CONFIG_MAP_NAME,
+        namespace: configMapNamespace,
+      })
+        .then((result: SelfSubjectAccessReviewKind) => {
+          if (result?.status?.allowed) {
+            setImportAllowed(result?.status?.allowed);
+          }
+        })
+        .catch(() => setImportAllowed(true));
```

Comment:
why do we return `true` on fail, what am I missing ?

Reply:
this is the default behaviour of rbac in console. If you fail checking permisions for something, then do it and fail/succeed a bit later

Reply:
can you point to an example ?

I found 2 places where we default to `false` and did not not find places where we default to `true`
[1] https://github.com/openshift/console/blob/e7565ea11aa80c80f8e0b34259f3be8fe56f8d57/frontend/packages/dev-console/src/components/import/image-search/ImageStreamNsDropdown.tsx#L38
[2] https://github.com/openshift/console/blob/aa4d040806db418f5348a998d5139fedaf27ae6e/frontend/packages/dev-console/src/utils/application-utils.ts#L351

Reply:
https://github.com/openshift/console/blob/master/frontend/public/components/utils/rbac.tsx#L119

Reply:
a - if you decide we should use this use case, you should also add the comment explaining the odd behaviour:
https://github.com/openshift/console/blob/master/frontend/public/components/utils/rbac.tsx#L114
b - this is not the default behaviour, `useAccessReview2` is used far less then `useAccessReview` that defaults to `false`
https://github.com/openshift/console/blob/master/frontend/public/components/utils/rbac.tsx#L127

Reply:
`useAccessReview` uses `useAccessReview2` under the hood, so it has the same behaviour and defaults to `true`

ok, copied the explanation there